deblur/test/test_workflow.py

```diff
@@ -52,6 +50,7 @@ def setUp(self):
 
         # the data directory for the workflow test files
         self.test_data_dir = join(dirname(abspath(__file__)), 'data')
+        self.seqs_fq_fp = join(self.test_data_dir, 'seqs.fq')
```

Comment:
Looks like this file wasn't committed to this PR, which is why the tests are failing